### DISCUSSION TITLE
Training and inference processes: GPU configuration, training time, number of epochs, etc.

### DISCUSSION THREAD

**Discussion by Claudia-Hello** [CederGroupHub/chgnet#141] [Q&A] [answered]
Hi,
I would like to ask CHGNet about the type and number of gpu's used in the training and inference process, the number of training data, the epoch of training, and the average training duration respectively.
Thank you！

**Comment by bowen-bd** [comment, marked as answer]
A single A100 gpu was used for training, for other details see: [chgnet/pretrained/0.3.0](https://github.com/CederGroupHub/chgnet/tree/fe0ad9a69b5378b6d9300b0d243c13d3c28a2b03/chgnet/pretrained/0.3.0)
